models/user.py

```diff
@@ -1,38 +1,67 @@
-from models.BaseModel import BaseModel
+from utility.helpers import query
+from utility.decor import path_set
+from flask import g
 
 
-class User(BaseModel):
-
+####
+class User():
     def __init__(self):
-        self.email_userid = {}
-        super().__init__()
+        self.path='SQL/user'
 
-    def add(self, entry):
-        userid = super().add(entry)
-        email = entry['email']
-        self.email_userid[email] = userid
+    def add(self, client_name, email, password):
+        query(self.path, 'register',
```

Comment:
Baba what's the problem with `query('user/register', vals=(...`? You don't even need the `SQL/` prefix everywhere.